Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "thread-id"
-version = "3.2.0"
+version = "3.2.1"
```

Comment:
Could you please avoid touching this? I bump the version just before making the release, not after making a change.